util/buildRelease/gen_release

```diff
@@ -166,6 +166,7 @@ SystemOrDie("make man-chpldoc");
 SystemOrDie("make clobber");
 
 print "Creating the examples directory...\n";
+SystemOrDie("rm -rf examples"); # remove examples symbolic link
```

Comment:
rm -rf <target> would happily rm a real (non-symlink) dir. 
rm -f <target> will happily rm a symlink, but will err if target is a real dir
Depends if you want to err on the side of accuracy, or on the side of clobbering, no matter what